packages/lab/src/__tests__/__e2e__/slider/Slider.cy.tsx

```diff
@@ -1,202 +1,169 @@
 import { Slider } from "@salt-ds/lab";
 
-describe("Given a Slider with a single value", () => {
-  it("THEN it should have ARIA roles and attributes", () => {
-    cy.mount(
-      <Slider
-        label={"TestLabel"}
-        min={5}
-        max={125}
-        step={5}
-        pageStep={25}
-        defaultValue={100}
-      />,
-    );
-    cy.findByRole("slider")
-      .should("have.attr", "aria-valuemin", "5")
-      .and("have.attr", "aria-valuemax", "125")
-      .and("have.attr", "aria-disabled", "false");
+describe("Given a Slider", () => {
```

Comment:
From what I can see we currently don't have a test for:
- min and max
- passing in a custom step vs not